samples/Plugin.Maui.Audio.Sample/ViewModels/AudioRecorderPageViewModel.cs

```diff
@@ -52,6 +55,84 @@ public bool IsRecording
 		this.dispatcher = dispatcher;
 	}
 
+	ChannelTypesViewModel selectedChannelType;
+	public ChannelTypesViewModel SelectedChannelType
+	{
+		get => selectedChannelType;
+		set
+		{
+			selectedChannelType = value;
+			NotifyPropertyChanged();
+		}
+	}
+	public ObservableCollection<ChannelTypesViewModel> ChannelTypes { get; set; } = new ObservableCollection<ChannelTypesViewModel>(Enum.GetValues(typeof(ChannelTypes)).Cast<ChannelTypes>().Select(x => new ChannelTypesViewModel()
+	{
+		ChannelType = x,
+		Name = x.ToString()
+	}).ToList());
+
+	BitDepthViewModel selectedBitDepth;
+	public BitDepthViewModel SelectedBitDepth
+	{
+		get => selectedBitDepth;
+		set
+		{
+			selectedBitDepth = value;
+			NotifyPropertyChanged();
+		}
+	}
+	public ObservableCollection<BitDepthViewModel> BitDepths { get; set; } = new ObservableCollection<BitDepthViewModel>(Enum.GetValues(typeof(BitDepths)).Cast<BitDepths>().Select(x => new BitDepthViewModel()
+	{
+		BitDepth = x,
+		Name = x.ToString()
+	}).ToList());
+
+
+	EncodingViewModel selectedEconding;
+	public EncodingViewModel SelectedEconding
+	{
+		get => selectedEconding;
+		set
+		{
+			selectedEconding = value;
+			NotifyPropertyChanged();
+		}
+	}
+
+	public ObservableCollection<EncodingViewModel> EncodingOptions { get; set; } =
+	[
+		new ()
+		{
+			Name = "LinearPCM",
+			Encoding = Encodings.LinearPCM,
+		},
+		new ()
+		{
+			Name = "Ulaw",
+			Encoding = Encodings.ULaw,
+		}
+	];
+
+
+	int selectedSampleRate = -1;
+	public int SelectedSampleRate
+	{
+		get => selectedSampleRate;
+		set
+		{
+			selectedSampleRate = value;
+			NotifyPropertyChanged();
+		}
+	}
+	public ObservableCollection<int> SampleRates { get; set; } =
```

Comment:
This is something I am not familiar with but are there a limited number of sample rates that we could/should supply? Or is it best to allow any value?

Reply:
I think in theory this should stay int, or maybe uint? Alternatively need to find all supported rates on any device, probably hard?
Actually BitDepth should maybe have more options too, 24 and 32 are also common, although I don't know if those are supported on the platforms.